hazelcast/include/hazelcast/client/address.h

```diff
@@ -15,10 +15,8 @@
  */
 #pragma once
 
-#include "hazelcast/util/hazelcast_dll.h"
```

Comment:
Isn't the `hazelcast_dll.h` header needed in this file? `HAZELCAST_API` is used down below.

Reply:
The next include `hazelcast/client/serialization/serialization.h` already includes it, hence less include lines.

Reply:
https://softwareengineering.stackexchange.com/questions/262019/is-it-good-practice-to-rely-on-headers-being-included-transitively

Reply:
OK, put back the includes.